vcf-dumper/vcf-dumper-lib/src/main/java/embl/ebi/variation/eva/vcfdump/VariantExporterController.java

```diff
@@ -54,44 +59,65 @@
 public class VariantExporterController {
 
     private static final Logger logger = LoggerFactory.getLogger(VariantExporterController.class);
-    private static final int WINDOW_SIZE = 20000;
+    private static final int WINDOW_SIZE = 10000;
 
     private final CellbaseWSClient cellBaseClient;
     private final String species;
     private final List<String> studies;
-    private final List<String> files;
-    private final String outputDir;
+    private Properties evaProperties;
+    private List<String> files;
+    private String outputDir;
     private final VariantSourceDBAdaptor variantSourceDBAdaptor;
     private final VariantDBAdaptor variantDBAdaptor;
     private final QueryOptions query;
     private final RegionFactory regionFactory;
     private final VariantExporter exporter;
+    private OutputStream outputStream;
     private Path outputFilePath;
     private int failedVariants;
+    private int totalExportedVariants;
+    private String outputFileName;
+
+    // TODO: remove dbName from this constructor after testing
```

Comment:
That's correct. The TODO message has been removed and the param is still in the constructors.